playwright/e2eTest.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { createUser, deleteUser } from './pages/CredentialPage.spec';
+import { createVfolder, deleteVfolder } from './pages/DataStoragePage.spec';
+import { login } from './pages/LoginPage.spec';
+import { createSession, deleteSession } from './pages/SessionPage.spec';
+import { test } from '@playwright/test';
+
+test('E2E test', async ({ page }) => {
```

Comment:
Currently, one test runs all tests. This is not a good pattern. Please separate them. You can google  e2e test examples that shows how to organize and separate tests.

Reply:
https://playwright.dev/docs/writing-tests#using-test-hooks